content/en/docs/tasks/configure-pod-container/user-namespaces.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Use a user namespace with a stateless pod
+reviewers:
+content_type: task
+weight: 160
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.25" state="alpha" >}}
+
+This page shows how to configure a user namespace for stateless pods. This
+allows to isolate the user running inside the container from the one in the
+host. A process running as root in a container can run as a different (non-root)
+user in the host.
+
+You can use this feature to reduce the damage a compromised container can do to
+the host or other pods in the same node. There are [several security
+vulnerabilities][KEP-vulns] rated HIGH and CRITICAL that were not explitable
+when user namespaces is active. It is expected user namespace will mitigate some
+future vulnerabilities too.
+
+[KEP-vulns]: https://github.com/kubernetes/enhancements/tree/217d790720c5aef09b8bd4d6ca96284a0affe6c2/keps/sig-node/127-user-namespaces#motivation
+
+## {{% heading "prerequisites" %}}
```

Comment:
There are some extra prerequisites:
- your cluster must include Linux nodes (Windows wouldn't work)
- you must either have permission to attach to Pods using `kubectl exec`, or shell access to the nodes. Consider mentioning this.

Reply:
Done, thanks